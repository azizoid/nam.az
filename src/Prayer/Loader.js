import React from "react";

const Loader = () => (
  <div className="text-center">
    <div className="lds-facebook">
      <div></div>
      <div></div>
      <div></div>
    </div>
  </div>
);

export default Loader;
