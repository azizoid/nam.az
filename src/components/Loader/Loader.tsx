import styles from './Loader.module.scss'

export const Loader = ():JSX.Element => (
  <div className="text-center">
    <div className={styles.ldsFacebook}>
      <div></div>
      <div></div>
      <div></div>
    </div>
  </div>
);
