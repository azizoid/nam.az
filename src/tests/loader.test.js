import { shallow } from "enzyme";

import React from "react";
import Loader from "../components/loader.component";

it("expect to render loader component", () => {
  const wrapper = shallow(<Loader />);
  expect(wrapper).toMatchSnapshot();
});
