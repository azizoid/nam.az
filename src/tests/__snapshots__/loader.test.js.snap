// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render loader component 1`] = `
<div
  className="text-center"
>
  <div
    className="lds-facebook"
  >
    <div />
    <div />
    <div />
  </div>
</div>
`;
